### PR TITLE
Button specs

### PR DESCRIPTION
**Pull request by LauRoxx**

### CODE REVIEW

File: packages/button/src/lib/IconButton.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  color?: "primary" | "secondary" | "tertiary" | "error";
+} & Omit<MUIIconButtonProps, 'aria-label' | 'color' | 'centerRipple' | 'disableRipple' | 'disableTouchRipple' | 'focusRipple' | 'disableFocusRipple' | 'TouchRippleProps' | 'touchRippleRef'>;
+
+
+
+export const IconButton = ({ children, title, filled, color = 'tertiary', ...rest }: IconButtonProps): JSX.Element => {
+  return (
+    <Tooltip title={title}>
+      { filled ?
+        <Button aria-label={title} color={color} startIcon={children} iconOnly {...rest}>{'\u2060'}</Button>
```
Comment (jordan-a-young) [inline packages/button/src/lib/IconButton.tsx:27]: Is there a prop people use to set the icon with this button?

**Review by jordan-a-young — CHANGES_REQUESTED** [review]

File: packages/button/src/lib/IconButton.tsx (hunk 1 above)
Comment (LauRoxx) [reply packages/button/src/lib/IconButton.tsx:27]: it's just children. This is likely going away since they changed what they wanted for iconbutton.

**Comment by LauRoxx** [comment]
test 
Screenshot 2023-05-15 at 2 32 18 PM

File: packages/theme/src/lib/light-theme.ts [hunk 2]
```diff
@@ -281,92 +281,59 @@ export const lightTheme = {
...
       },
       styleOverrides: {
-        root: {
-          minWidth: '128px',
-          textTransform: 'none',
-        },
+        root: ({ ownerState } : any) => ({
+          ...(ownerState.variant === 'contained' ?
+            {
+              backgroundColor: tokens[`color${ownerState.color.charAt(0).toUpperCase() + ownerState.color.slice(1)}Main` as keyof typeof tokens],
```
Comment (LauRoxx) [inline packages/theme/src/lib/light-theme.ts:287]: test 
Screenshot 2023-05-15 at 2 32 18 PM

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**